CHANGES.rst

```diff
@@ -5,7 +5,8 @@
 2.4.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Fix transaction manager run problems
```

Comment:
I think this needs to go to version 2.5.0 given the semantic changes.

I would like to see a more detailed change note, mentioning the `commit` change discussed below and the fact that the transaction is no longer captured by run but allowed to be changed while `run` is in progress. The `commit` change in particular is not mentioned anywhere in #58. (As a user, I'd prefer not to have to read the diff to get a reasonable understanding of potential behaviour changes; I'd like the change note to give a quick summary, and if there is anything that seems like it might impact me I'll follow up with the issue and/or PR.)

Reply:
I have made the `Changes.rst` description more explicit.